test/integration/scheduler_perf/test-performance.sh

```diff
@@ -48,5 +48,6 @@ if ${RUN_BENCHMARK:-false}; then
 fi
 # Running density tests. It might take a long time.
 kube::log::status "performance test (density) start"
-go test -test.run=. -test.timeout=60m -test.short=false
+go test -test.timeout=60m -test.short=false -test.nodes=100 -test.pods=3000 -test.run=.
```

Comment:
no args needed here, refactor `Test100Nodes3000Pods{}` to call a generic `func GenericTestImplementation(schedulerPerfConfig config)` function where `pods, nodes` are fields added to `schedulerPerfConfig`.